rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py

```diff
@@ -160,6 +160,20 @@ def test_exec(config, ssh_con):
             )
```

Comment:
Can we add config details in the documentation section..

Reply:
Done